packages/components/modal/modal.service.ts

```diff
@@ -9,6 +9,9 @@ import { KbqModalRef } from './modal-ref.class';
 import { KbqModalComponent } from './modal.component';
 import { ConfirmType, IModalOptionsForService, ModalOptions } from './modal.type';
 
+/** Injection token that can be used to access the data that was passed in to a modal. */
+export const KBQ_MODAL_DATA = new InjectionToken<any>('KbqModalData');
```

Comment:
```suggestion
export const KBQ_MODAL_DATA = new InjectionToken<unknown>('KbqModalData');
```
по хорошему бы в таких местах `unknown` использовать (вкусовщина)
в дальнейшем можно будет включить `noImplicitAny` (мечты) :)